MobileBuy/buy3/src/main/java/com/shopify/buy3/GraphClient.kt

```diff
@@ -257,14 +272,16 @@ private fun OkHttpClient.withHttpCacheInterceptor(httpCache: HttpCache?): OkHttp
     }
 }
 
-private fun OkHttpClient.withSdkHeaderInterceptor(applicationName: String, accessToken: String): OkHttpClient {
+private fun OkHttpClient.withSdkHeaderInterceptor(applicationName: String, accessToken: String, locale: String?): OkHttpClient {
     return newBuilder().addInterceptor { chain ->
         val original = chain.request()
         val builder = original.newBuilder().method(original.method(), original.body())
+        val usedLocale = locale ?: ""
```

Comment:
if we are passing null all the way down to here, do we need this line or we can just set the header to null?